R/data_copy.R

```diff
@@ -65,7 +65,7 @@ spark_serialize_csv_string <- function(sc, df, columns, repartition) {
   }), optional = TRUE)
 
   tempFile <- tempfile(fileext = ".csv")
-  write.table(df, tempFile, sep = "\31", col.names = FALSE, row.names = FALSE, quote = FALSE)
```

Comment:
`\31` is not a valid json character that we use in livy, need to take a look a this a bit more and scope to this fix to livy only or find a valid separator for both, sockets and json.